transcoder/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+file(GLOB BASISU_TRANSCODER_HDRS ${CMAKE_CURRENT_SOURCE_DIR}/*.h ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+file(GLOB BASISU_TRANSCODER_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+
+add_library(basisu_transcoder ${BASISU_TRANSCODER_SRCS} ${BASISU_TRANSCODER_HDRS})
+target_include_directories(basisu_transcoder
+    PUBLIC
+    ${CMAKE_CURRENT_SOURCE_DIR}
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/..>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
```

Comment:
It's nice you're using `BUILD_INTERFACE` and `INSTALL_INTERFACE` generator expressions.
But they are not really required because you're not installing the exported targets using `install(EXPORT)`